honeybee_doe2/writer.py

```diff
@@ -15,8 +15,21 @@
 from honeybee.typing import clean_string
 
 
-def model_to_inp(hb_model):
+def model_to_inp(hb_model, hvac_grouping):
     # type: (Model) -> str
+    """
+        args:
+            hb_model: Honeybee model
+            hvac_grouping: accepts: 'room', 'story', 'model'
+    """
+    hvac_groups = None
+    if hvac_grouping == 'room':
+        hvac_groups = hb_model.properties.doe2.hvac_sys_zones_by_room
+    elif hvac_grouping == 'story':
+        hvac_groups = hb_model.properties.doe2.hvac_sys_zones_by_story
+    elif hvac_grouping == 'model':
+        hvac_groups = hb_model.properties.doe2.hvac_sys_zones_by_model
+
```

Comment:
What if the user uses a wrong input? You need a final check to raise an error for an invalid input.

Reply:
done! [see commit](https://github.com/ladybug-tools/honeybee-doe2/pull/59/commits/52b45eb0190036830d0e1645c327ca26d406b2b6)